src/commands/moderation/createrole.js

```diff
@@ -0,0 +1,33 @@
+const { Command, SwitchbladeEmbed, Constants } = require('../../')
+
+module.exports = class CreateRole extends Command {
+  constructor (client) {
+    super(client, {
+      name: 'createrole',
+      category: 'moderation',
+      requirements: { guildOnly: true, botPermissions: ['MANAGE_ROLES'], permissions: ['MANAGE_ROLES'] },
+      parameters: [
+        { type: 'color', missingError: 'commands:createrole.noParams' },
```

Comment:
maybe color parameter optional (see currency command for example)

Reply:
Done!